signxml/xades/__init__.py

```diff
@@ -326,12 +338,9 @@ def _generate_xades_ssp_elements(self, options_struct):
                 encoding of one DER-encoded instance of type IssuerSerial type
                 defined in IETF RFC 5035 [17].
                 """
-                raise NotImplementedError(
-                    "Please make a PR if you know how to obtain in python a "
-                    "'DER-encoded instance of type IssuerSerial type defined in "
-                    "IETF RFC 5035'")
                 serial_element = XADES132.IssuerSerialV2(
-                    # TODO implement, wtf?
+                    DS.X09IssuerName(issuer_string(cert)),
+                    DS.X509SerialNumber(f"cert.get_serial_number()")
```

Comment:
Not quite yet: `<xsd:element name="IssuerSerialV2" type="xsd:base64Binary" minOccurs="0"/>`
The whole ASN.1 object shall be DER encoded and put here as bytestring.

Reply:
I compared the result from XML signed (xades-bes) and what `get_serial_number()` it's ok.
which schema file got it ?

Reply:
I think you implemented the [legacy IssuerSerial](https://github.com/xoe-labs/signxml/blob/fd15c364e46946569a7bf75318b8adc77e9d1ae5/signxml/xades/schemas/v1.3.2/XAdES01903v132-201601.xsd#L238)

See the [schema of IssuerSerialV2](https://github.com/xoe-labs/signxml/blob/fd15c364e46946569a7bf75318b8adc77e9d1ae5/signxml/xades/schemas/v1.3.2/XAdES01903v132-201601.xsd#L259)

Reply:
Nah, the object itself is der encoded, not one of it's members...